app/api/scraper.py

```diff
@@ -54,3 +60,43 @@ def user_detail_check(
         score=prediction.score,
     )
     return response
+
+
+@router.get("/export", name="user:export-detail")
+def export_profile(
+    url: str, bg_tasks: BackgroundTasks, scraper: TwitterScraper = Depends()
+):
+    if url == "":
+        raise HTTPException(400, "'url' argument is invalid!")
+
+    username = get_twitter_username(url)
+    full_details = scraper.get_full_detail(username, 3)
+
+    file_path = f"{full_details.screen_name}.json"
+    with open(file_path, "w") as f:
+        full_details_str = (
+            str(full_details._json)
+            .replace('"', '\\"')
+            .replace("'", '"')
+            .replace(': \\"', ': "')
+            .replace('\\",', '",')
+            .replace("None", "null")
+            .replace("True", "true")
+            .replace("False", "false")
+        )
+        full_details_str = re.sub(r'(\w)"(\w)', r"\1'\2", full_details_str)
```

Comment:
all of this is me trying to solve all the cases when converting from `Tweepy._json` to valid string for `json.loads()`. You can export "JohnCena" with `tweets_num=3` to see all the errors:
* **line 79, 80**: Swap `"` with `'`
  * Ex: `Tweepy._json`: `{'key': 'value "a" here'}` but `json.loads()` need: `{"key": "value 'a' here"}`
* **line 81, 82**: Specific solution for stupid case
  * Ex:  `Tweepy.Tweet._json`: `{'key': "value 'a' here", ...}`. <- (See it with "JohnCena" without line 81,82)
* **line 83,84,85**: yea, it can't read the left one :V
* **line 87**: Solution for "I'm, he's, she's, they're, ...". Can't keep it safe from **line 80**, so here is the fix.

Reply:
Why do you convert a Python dictionary to string and `json.loads` it again? I don't understand what you are trying to do here.

Reply:
the pure python dictionnary look awful when exporting, so i use `json.dumps(mydata, indent=4)` to make it prettier

Reply:
the pure python dictionnary look awful when exporting, so i use `json.dumps(mydata, indent=4)` to make it prettier

Reply:
On Slack please

Reply:
📞🚑🚨!!

Reply:
@dirtyg2120 Why not just return dictionary directly and let frontend stringify it?

Reply:
> @dirtyg2120 Why not just return dictionary directly and let frontend stringify it?

so export just need to response a dictionary and frontend will handle the download file huh?